MatrixGame/src/MatrixGame.cpp

```diff
@@ -546,7 +506,7 @@ void MatrixGameInit(HINSTANCE inst, HWND wnd, wchar *map, SRobotsSettings *set,
     // if (iface_save) bpi.SaveInTextFile(IF_PATH, true);
 }
 
-void SRobotsSettings::ApplyVideoParams(void) {
+void CGame::ApplyVideoParams(SRobotsSettings *set) {
```

Comment:
maybe consider using reference instead of a pointer here.

Reply:
Looks good